contracts/staking/src/execute.rs

```diff
@@ -786,6 +812,12 @@ pub fn update_config(
         config.oracle_contract_address = Some(Addr::unchecked(oracle_contract_address));
     }
 
+    if oracle_contract_address_v2.is_some() {
+        let oracle_contract_address_v2 = oracle_contract_address_v2.unwrap();
+        validate_address(&oracle_contract_address_v2, "osmo")?;
+        config.oracle_contract_address_v2 = Some(Addr::unchecked(oracle_contract_address_v2));
```

Comment:
The same as above also applies here: 

```suggestion
        let address = validate_address(&oracle_contract_address_v2, "osmo")?;
        config.oracle_contract_address_v2 = Some(address);
```